main.js

```diff
@@ -0,0 +1,8 @@
+document.querySelector("header nav li").remove(); //removes 'all posts'
+document.querySelector(".featured span").remove();//removes 'read time'
+const allPosts = document.querySelectorAll("main .posts article");
+allPosts[2].remove(); // removes last post
+document.querySelector("main .posts h3").remove(); //removes first title
+document.querySelector("main .posts h3").remove(); //removes second title
```

Comment:
```suggestion
document.querySelectorAll("main .posts h3").forEach((el) => el.remove());
```
the reason that you were getting an error when you tried `document.querySelectorAll("main .posts h3").remove()` is because `querySelectorAll` returns a **_node list_**. A node list is similar to an array, and .remove does NOT work on entire node lists. Instead, you have to iterate over the node list and delete each element one at a time.